src/widgets/Label.cpp

```diff
@@ -127,30 +148,85 @@ void Label::paintEvent(QPaintEvent *)
     {
         option.setWrapMode(QTextOption::NoWrap);
     }
-    painter.drawText(textRect, this->text_, option);
+    painter.drawText(textRect, text, option);
 
 #if 0
-    painter.setPen(QColor(255, 0, 0));
-    painter.drawRect(0, 0, this->width() - 1, this->height() - 1);
+        painter.setPen(QColor(255, 0, 0));
+        painter.drawRect(0, 0, this->width() - 1, this->height() - 1);
 #endif
 }
 
+void Label::resizeEvent(QResizeEvent *event)
+{
+    if (this->shouldElide_)
+    {
+        auto metrics = this->getFontMetrics();
+        if (this->updateElidedText(metrics, this->getWidthWithoutOffset()))
+        {
+            this->update();
+        }
+    }
+
+    BaseWidget::resizeEvent(event);
+}
+
+QFontMetricsF Label::getFontMetrics() const
+{
+    return getApp()->getFonts()->getFontMetrics(this->fontStyle_,
+                                                this->scale());
+}
+
+qreal Label::getWidthWithoutOffset() const
+{
+    if (this->hasOffset_)
+    {
+        return this->width() - this->getOffset() - this->getOffset();
+    }
+
+    return this->width();
```

Comment:
```suggestion
    return this->width() - (2 * this->getOffset());
```

Reply:
resolved in [`d21587c` (#6318)](https://github.com/Chatterino/chatterino2/pull/6318/commits/d21587c605d24a7b2d6e3738d567acbfb03fa69d)